styled/auth.js

```diff
@@ -1,33 +1,87 @@
 import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
 
-export const SignUpContainer = styled.View`
+// TODO @tommypoa: Consider creating re-usable containers taking in props for the 3 Auth screens
+
+// StartUpScreen
+
+export const StartUpContainer = styled.View`
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: #fff;
+  margin-top: 241px;
 `;
 
-export const Input = styled.TextInput`
-  width: 350px;
-  height: 55px;
-  background-color: #008550;
-  margin: 10px;
-  padding: 8px;
-  border-radius: 14px;
-  color: white;
-  font-size: 18px;
-  font-weight: 500;
-  align-content: center;
+export const StartUpTitleContainer = styled.View`
+  margin-left: 58px;
+  margin-right: 47px;
+`;
+
+export const StartUpSignInContainer = styled.View`
+  margin-top: 108px;
 `;
 
+export const StartUpLogInContainer = styled.View`
+  margin-top: 16px;
+  margin-left: 42px;
+  margin-right: 47px;
```

Comment:
Like I mentioned before, avoid pixels for centering. Try `margin-left: 4%;
  margin-right: 4%;` (the margins we use for stores and product screens I believe) or something like that and keep it standard across these containers.